docs/source/principles.md

```diff
@@ -47,22 +47,23 @@ TREs holding public data should build and maintain the trust of data subjects an
 
 ### Rationale
 
-Maintaining trust in TREs to hold and use data is essential to prevent backlash or resistance to the use of data.
-This can include maintaining the trust of members of the public who are impacted by research, as well as the trust of commercial data providers.
+To ensure continued public support for the use of data and to alleviate possible concerns, it is vital to maintain public trust in the ways TREs hold and use data.
```

Comment:
I think be explicit that we want public support both for use of their data and the use of TREs to access that data.

Could we be slightly more specific about "possible concerns"?  What kind of concerns? Ethical use of the data? Security concerns such as making sure only the "right" people have access?


```suggestion
To ensure continued public support both for the use of data and TREs to facilitate research, it is vital to maintain public trust in the ways TREs hold and use data and to alleviate possible concerns.
```